server/app/models/place.js

```diff
@@ -16,7 +16,7 @@ var PlaceSchema = new Schema({
     required: true
   },
   photos: [String],
-  likes: [Schema.Types.ObjectId],
+  likes: [String],
```

Comment:
Maybe you're right, but we discuss about it. And we decided, that there should be only id. And in future we'll not need to know about userName or something else in likes. But if you want, I'll do it :)